packages/markdown/src/MarkdownEditor.tsx

```diff
@@ -152,18 +161,9 @@ export function MarkdownEditorConnected(props: MarkdownEditorProps) {
       field={props.sdk.field}
       isInitiallyDisabled={props.isInitiallyDisabled}
     >
-      {({ value, disabled, setValue, externalReset }) => {
```

Comment:
Regarding [`externalReset logic](https://github.com/contentful/field-editors/blob/3ae9c46f02565c17a9323b1d319950bbb5adc900/packages/_shared/src/FieldConnector.ts#L116): This might be important to reset the editor whenever a remote change (other user, CMA change) happens.

If you remove it, the editor might not properly pick up on remote changes, or there could be certain glitches, leading to the remote change being overwritten again. Not quite sure.

The question is though: If the user experiences a focus loss issue, but there's no other user editing remotely, why is the external reset triggered?

Reply:
Through the auto-save of the editor if you type during it it comes to this glitch

With these changes it works and I could also tried the update magic with two different tabs and it looks also fine

Reply:
On other Editors (RichText, SingleLine) we also don't have this `externalReset`
RichTextEditor uses basically the same approach as this one

Reply:
@chrishelgert let me know if you want to test two different users making edits in the same markdown field

Reply:
@DanweDE Did some small adjustments and tested it together with @YvesRijckaert 
It works if we both work on the same editor correctly